src/main/kotlin/org/jitsi/rtp/util/RtpUtils.kt

```diff
@@ -61,10 +64,37 @@ class RtpUtils {
 
         fun getNumPaddingBytes(dataSizeBytes: Int): Int {
             var paddingBytes = 0
-            while (dataSizeBytes + paddingBytes % 4 != 0) {
+            while ((dataSizeBytes + paddingBytes) % 4 != 0) {
                 paddingBytes++
             }
             return paddingBytes
         }
+        /**
+         * Returns the delta between two RTP sequence numbers, taking into account
+         * rollover.  This will return the 'shortest' delta between the two
+         * sequence numbers in the form of the number you'd add to b to get a. e.g.:
+         * getSequenceNumberDelta(1, 10) -> -9 (10 + -9 = 1)
+         * getSequenceNumberDelta(1, 65530) -> 7 (65530 + 7 = 1)
+         * @return the delta between two RTP sequence numbers (modulo 2^16).
+         */
+        fun getSequenceNumberDelta(a: Int, b: Int): Int
+        {
+            val diff = a - b;
```

Comment:
`RtpSequenceNumber.kt` has the same calculation. Can we have it in one place (I'm not sure what's the best way to reuse, perhaps have `RtpSequenceNumber.kt` call into this as a utility)?